### PR TITLE
CloneHeroTeam.CloneHero version 1.0.0.4080

### PR DESCRIPTION
**Pull request by stuffnthingzz**
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [ ] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`?
- [ ] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.2 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.2.0)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/93274)

### CODE REVIEW

**Comment by dylanh12210** [comment]
@microsoft-github-policy-service agree [company="{stuff}"]

**Comment by dylanh12210** [comment]
@microsoft-github-policy-service agree

File: manifests/c/CloneHeroTeam/CloneHero/1.0.0.4080/CloneHeroTeam.CloneHero.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,13 @@
+# Created using wingetcreate 1.1.2.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.2.0.schema.json
+
+PackageIdentifier: CloneHeroTeam.CloneHero
+PackageVersion: 1.0.0.4080
+Installers:
+- Architecture: x64
+  InstallerType: inno
+  InstallerUrl: https://pubdl.clonehero.net/clonehero-v1.0.0.4080-final/CloneHeroSetup-v1.0.0.4080-final-win64.exe
+  InstallerSha256: 54E730EEFAF5211095A4535995491EC2693AB60B6E9A2E4238A6B36E33A7B626
```
[the remark this answers is not in the document]
Comment (stephengillie) [reply manifests/c/CloneHeroTeam/CloneHero/1.0.0.4080/CloneHeroTeam.CloneHero.installer.yaml:10]: This clears the installation prompt: 
image

**Comment by ItzLevvie** [comment]
Close with reason: Stale pull request;

**Pull request closed without merging**